custom/icds_reports/templates/icds_reports/icds_app/kpi.directive.html

```diff
@@ -1,5 +1,5 @@
 <div class="row" ng-repeat="section in $ctrl.data track by $index">
-    <div class="col-md-6" ng-repeat="cell in section track by $index">
+    <div ng-if="((cell.label !== 'AWCs Reported Infantometer' && cell.label !== 'AWCs Reported Stadiometer' && cell.label !== 'Infantometer' && cell.label !== 'Stadiometer') || $ctrl.haveAccessToFeatures)" class="col-md-6" ng-repeat="cell in section track by $index">
```

Comment:
@challabeehyv I think this hard-coding of display logic against the labels is quite dangerous. I'd like to add a `show_in_kpis` field to this data model instead so it's more explicit and future proof. Can you tell me where I can go to test this is working properly? (e.g. where these are currently being hidden)?

Reply:
understood. I thought the feature flag is more like a temporary thing for testing purpose. I will make the change. 
https://india.commcarehq.org/a/icds-dashboard-qa/icds_dashboard/#/program_summary/awc_infrastructure
https://india.commcarehq.org/a/icds-dashboard-qa/icds_dashboard/#/awc_reports/awc_infrastructure?location_name=Basinapalli%20%5B28553020101%5D&location_id=0090df1176b2484d82b0d857c4697123&selectedLocationLevel=4
https://india.commcarehq.org/a/icds-dashboard-qa/icds_dashboard/#/awc_infrastructure/infantometer/map?location_name=C.K.Palli%20%5B285530201%5D&location_id=90be6e6a07f9433ab5fae105099d2e53&selectedLocationLevel=3
https://india.commcarehq.org/a/icds-dashboard-qa/icds_dashboard/#/awc_infrastructure/stadiometer/map?location_name=C.K.Palli%20%5B285530201%5D&location_id=90be6e6a07f9433ab5fae105099d2e53&selectedLocationLevel=3

Reply:
Sorry, you're saying this check will be deleted once the report is released? Then I feel less concerned (though if it's a simple change then great if you can make it).

Reply:
yes. This is for testing only. Anyways i will make the change